### ISSUE TITLE
Multiple remote on same 4CH

### ISSUE THREAD

**Issue by jmazier-j2d** [iMarkus/Sonoff-Somfy-HomeAssistant#2]
Hi Markus,

Nice work ! I'm really looking into applying the exact same solution.
Quick question tho, would that be possible to plug 3 remote on the same 4CH, for sure that would not be possible to control each one independently, but would that work ? 😅

Thanks a lot !

**Comment by iMarkus** [comment]
Hi,
thanks a lot. I think it should work as soon as you take the 4 ch pro. 

Cheers

**Comment by jmazier-j2d** [comment]
I'll give a try. Worth case scenario, I'll need to order 2 more 4CH pro.
I'll keep you updated, could be interesting for others.

**Comment by jmazier-j2d** [comment]
Hi,

So I'm finally done with everything. Just to let you know : It's not working as expected. After plugging all the remotes, they were all triggered at the same time and I think that was causing an issue (because all signal were sent at the exact same time, maybe ?)
So what I've done, is configuring 1 remote to control all blinds and plugging this remote to the 4CH. I've used the Raven firmware to be able to use it with Homekit and it's working like a charm (combined with Siri shortcuts, I was able to configure different scenarios like "half opened" and "half closed").

Lastly, I've lost 1 remote in the process, I fried one button while welding, I think I fried one button due to the heat... But hopefully, I had 3 of them, so I'm all good now !
